src/OSPSuite.Core/Domain/Data/SimulationResults.cs

```diff
@@ -132,6 +132,17 @@ public virtual IReadOnlyList<int> AllIndividualIds()
          }
       }
 
+      public virtual int NumberOfIndividuals
+      {
+         get
+         {
+            var allIndividualsIds = AllIndividualIds();
+            //+1 because individual ids are typically 0 based
+            var maxNumberOfIndividuals = allIndividualsIds.Any() ? allIndividualsIds.Max() + 1 : 0;
+            return Math.Max(Count, maxNumberOfIndividuals);
```

Comment:
Just in case the last items did not work, count could be >maxNumberOfIndividuals . we take the max to be sure